### ISSUE TITLE
Troubleshooting [test_simple.py]

### ISSUE THREAD

**Issue by rnahumaf** [nianticlabs/monodepth2#253]
Hi, I'm begginer in this area, and I installed the package in my anaconda from windows. Although this, I can't used the first code:

`python test_simple.py --image_path assets/test_image.jpg --model_name mono+stereo_640x192`

This is the message with error:

`(monodepth2) C:\Users\betag>python test_simple.py --image_path assets/test_image.jpg --model_name mono+stereo_640x192
Traceback (most recent call last):
  File "test_simple.py", line 21, in <module>
    import networks
ModuleNotFoundError: No module named 'networks'`

Can anybody help me?

Thanks

**Comment by mdfirman** [comment]
Hi – did you move `test_simple.py` away from its original location? 

After downloading the code, you should have a folder `monodepth2`. Inside this folder there should be several files and folders – including a file `test_simple.py` and a folder called `networks`.

If you then navigate to the `monodepth2` folder and run your python command, you shouldn't see the error you have above.

(I will confess though I don't have any familiarity with running python on windows so I am basing this answer on how things work on Unix)

**Comment by rnahumaf** [comment]
@mdfirman Thanks for your quick reply! In fact, I can't tell you what went wrong exactly, but I guess it had something to do with the proper installation of modules. After I did `conda install torchvision -c pytorch` and `conda install PyTorch -c PyTorch `, I was able to successfully run `python test_simple.py --image_path assets/test_image.jpg --model_name mono+stereo_640x192` and get the desired output in the /assets folder.

Thanks!
